spec/support/schemas/claims_api/v2/veterans/claims/claim.json

```diff
@@ -37,6 +37,11 @@
       "description": "Date claim was closed",
       "example": "2019-09-04"
     },
+    "currentPhaseBack": {
+      "type": "boolean",
+      "nullable": true,
```

Comment:
I'm not sure we want this to be nullable. 🤔 
I believe it should be either `true` or `false`.